translator/reader/reader.cpp

```diff
@@ -250,13 +250,16 @@ void UsdArnoldReader::TraverseStage(UsdPrim *rootPrim, UsdArnoldReaderContext &c
             UsdGeomImageable imageable(prim);
             bool pruneChildren = false;
             attr = imageable.GetVisibilityAttr();
-            if (attr && attr.HasAuthoredValue())
+            if (attr && attr.HasAuthoredValue()) {
+
                 pruneChildren |= (attr.Get(&visibility, frame) && 
                         visibility == UsdGeomTokens->invisible);
+            }
 
             attr = imageable.GetPurposeAttr();
             if (attr && attr.HasAuthoredValue()) {
                 pruneChildren |= ((attr.Get(&purpose, frame) && 
+                        !purpose.IsEmpty() && 
```

Comment:
it looks like the only allowed values are "default, render, proxy, guide", so the scene shouldn't have an authored empty purpose. Testing it is ok, but it might be the case where the scene with empty purpose won't work with other usd compatible applications or will have other meaning. 
Should we also add a warning ligne in debug mode ? just in case we have to debug scenes in the future ?

Reply:
actually, the reason we're doing this change is because other tools (usd_view, mayausd, etc..) are showing these primitives.
It might not be intentional, as the docs don't mention it, but at least we'll now be consistent.